libs/barista-components/formatters/src/date/date-range.spec.ts

```diff
@@ -39,7 +43,7 @@ import {
 
 describe('DtDateRange', () => {
   let pipe: DtDateRange;
-  let spiedDate: jest.SpyInstance;
+  let spiedDate: any; // jest.SpyInstance;
```

Comment:
Had to type it as any as Typescript compilation failed:

```
ERROR: /Users/lukas.holzer/Sites/barista/libs/barista-components/formatters/BUILD.bazel:45:5: Compiling TypeScript (devmode) //libs/barista-components/formatters:test_compile failed (Exit 1)
libs/barista-components/formatters/src/date/date-range.spec.ts:46:23 - error TS2694: Namespace 'jest' has no exported member 'SpyInstance'.

46   let spiedDate: jest.SpyInstance;
                         ~~~~~~~~~~~

Target //libs/barista-components/formatters:test failed to build
```


Very strange as the type declaration file in `node_modules/@types/jest/index.d.ts` has the interface on line 1055.

```typescript

declare namespace jest {
    interface SpyInstance<T = any, Y extends any[] = any> extends MockInstance<T, Y> {}
}
```